framework/source/class/qx/ui/basic/Image.js

```diff
@@ -753,6 +753,11 @@ qx.Class.define("qx.ui.basic.Image",
       if (isFont) {
         var size;
 
+        // Don't use scale if size is set via postfix
+        if (this.getScale() && parseInt(source.split("/")[2])) {
```

Comment:
always a good idea to specify the radix as the second parameter when using parseInt, so no one is fooled by leading leading zeros meaning octal or other such silliness